app/layout/header/component.tsx

```diff
@@ -2,20 +2,24 @@ import React from 'react';
 
 import Link from 'next/link';
 
-import Wrapper from 'layout/wrapper';
-import User from 'layout/header/user';
+import { useMe } from 'hooks/me';
+
+import classnames from 'classnames';
+
 import MyProjects from 'layout/header/my-projects';
 import Title from 'layout/header/title';
+import User from 'layout/header/user';
+import Wrapper from 'layout/wrapper';
 
 import LinkButton from 'components/button';
 
-import { useMe } from 'hooks/me';
-
+import LOGO_BLACK from 'svgs/logo-black.svg';
```

Comment:
Do we need a LOGO BLACK? maybe we can directly change the fill depending on the theme